packages/core/README.md

```diff
@@ -1,3 +1,396 @@
+
+**Important:** Your main screen's route should be called `"Root"`. This is the screen we will
+navigate to once login/whatsnew/onboarding is finished. If this is a limitation in your app, please
+create an issue!
+
+`SettingsScreen` also has to be added manually. This is because you most likely have app-specific
+settings you want to hook up to the settings screen.
+
+If you don't have any app-specific settings, you can just add `SettingsScreen` from
+`@equinor/mad-core` with name `"Settings"`. If you do have app-specific settings you want to add, we
+suggest creating a wrapper component that passes in the props you need to `SettingsScreen`.
+
+Example stack:
+
+```tsx
+const CoreStack = createCoreStackNavigator<RootStackParamList>(config);
+function RootNavigator() {
+    return (
+        <CoreStack.Navigator>
+            <CoreStack.Screen
+                name="Root"
+                component={BottomTabNavigator}
+                options={{ headerShown: false }}
+            />
+            <CoreStack.Screen
+                name="NotFound"
+                component={NotFoundScreen}
+                options={{ title: "Oops!" }}
+            />
+            <CoreStack.Screen name="Settings" component={SampleSettingsScreen} />
+        </CoreStack.Navigator>
+    );
+}
+```
+
+SampleSettingsScreen:
+
+```tsx
+import React from "react";
+import { SettingsScreen, SettingsScreenConfiguration } from "@equinor/mad-core";
+
+export const SampleSettingsScreen = () => {
+    const appSpecificSettingsConfig: SettingsScreenConfiguration = [
+        {
+            items: [
+                {
+                    name: "navigation",
+                    title: "navigation",
+                    onPress: () => undefined,
+                    iconName: "abacus",
+                },
+                {
+                    name: "button",
+                    title: "Button",
+                    onPress: () => undefined,
+                    iconName: "abacus",
+                    color: "primary",
+                },
+                {
+                    name: "switch",
+                    title: "Test",
+                    onChange: () => undefined,
+                    isActive: true,
+                    iconName: "abacus",
+                },
+            ],
+        },
+    ];
+
+    return <SettingsScreen config={appSpecificSettingsConfig} />;
+};
+```
+
+#### Step 4: replace `createStackNavigator`/`createNativeStackNavigator`/`createBottomTabsNavigator`/`NavigationContainer` imports
+
+In order for us to display environment banners and service messages on all screens, you need to
+import `createNativeStackNavigator`/`createBottomTabsNavigator` from `@equinor/mad-core` instead of
+`react-navigation`. Currently we don't support other navigators. Do you need support for other
+navigators? Create an issue!
+
+By default environment-banners and service-messages should display if the screen header is
```

Comment:
With dash in `environment-banners` and `service-messages` here and without in the previous paragraph. I'd stick to the spelling in the first paragraph.

Reply:
Fixed